tensorflow/python/ops/math_ops.py

```diff
@@ -2109,6 +2110,14 @@ def reduce_variance(input_tensor, axis=None, keepdims=False, name=None):
   """
   name = name if name else "reduce_variance"
   with ops.name_scope(name):
+    if isinstance(input_tensor, ragged_tensor.RaggedTensor):
+      square_of_input = gen_math_ops.square(input_tensor)
+      mean_of_square = reduce_mean(square_of_input,
+      	                           axis=axis,
+      	                           keepdims=keepdims)
```

Comment:
Make sure you run pylint, the indentation on these lines looks strange

Reply:
done